src/main/java/com/mmaarten/mmcommands/MmCommandHandler.java

```diff
@@ -772,13 +501,15 @@ protected boolean noPermission(@NotNull CommandSender sender, @NotNull MmCommand
 
     /**
      * Print the help menu to the provided commandsender.
-     * The help menu consists of the {@link #helpHeader(String) header}
-     * followed by {@link #amountOfCommandsPerHelpPage(int) a specified amount}
+     * The help menu consists of the {@link MmCommandHandler#helpHeader header}
+     * followed by {@link MmCommandHandler#commandsPerPage a specified amount}
      * of help entries.
      * Each entry follows this pattern:
-     * {@link #helpCommandPrefix(String) commandPrefix} {@code /label [subcommand(s)]}
-     * {@link #helpArgumentSpacer(String) argumentSpacer} {@link MmCommandSignature#arguments() arguments}
-     * {@link #helpDescriptionSpacer(String) descriptionSpacer} {@link MmCommandSignature#description() description}
+     * {@link MmCommandHandler#helpCommandPrefix commandPrefix} {@code /label [subcommand(s)]}
+     * {@link MmCommandHandler#helpCommandArgumentSpacer argumentSpacer}
+     * {@link MmCommandSignature#arguments() arguments}
+     * {@link MmCommandHandler#helpArgumentDescriptionSpacer descriptionSpacer}
+     * {@link MmCommandSignature#description() description}
```

Comment:
Please make sure to always reference methods in javadoc. Fields will become private. (please check this in all edited javadoc)